core/src/components/datetime/readme.md

```diff
@@ -147,18 +147,36 @@ const zonedTime = dateFnsTz.utcToZonedTime(date, userTimeZone);
 format(zonedTime, 'yyyy-MM-dd HH:mm:ssXXX', { timeZone: userTimeZone });
 ```
 
-## Parsing Dates
+## Timezones
```

Comment:
Can we explicitly call out that ion-datetime does not manipulate/read timezone? Meaning users need to pass in an ISO-8601 string that is already configured for the user's timezone if passing a set value.

If there is no value, then we default to the time specified on a user's machine (which will already be in the user's timezone)